src/utils/functions/chat.ts

```diff
@@ -0,0 +1,47 @@
+import { MentionData } from 'react-native-controlled-mentions/dist/types';
+
+import { ChatMessage, DiscriminatedItem } from '@graasp/sdk';
+
+import {
+  MENTION_CHAT_TRIGGER,
+  MENTION_REGEX_WITH_NAME,
+} from '../../config/constants/constants';
+import {
+  ITEM_NAVIGATOR,
+  ITEM_NAVIGATOR_ITEM_CHAT,
+} from '../../navigation/names';
+import { ItemScreenProps } from '../../navigation/types';
+
+export const convertToGiftedMessages = (originalChat: ChatMessage[]) => {
+  return originalChat.map(({ id, body, createdAt, creator }) => ({
```

Comment:
No need to use the return keyword a function body. Refer to [the `arrow-body-style` with the "as-needed" value Eslint rule](https://eslint.org/docs/latest/rules/arrow-body-style#as-needed)